pkg/billing/client.go

```diff
@@ -35,16 +35,138 @@ func GetHostname() string {
 	return hostname
 }
 
-func NewClient(url string, c *http.Client) Client {
-	return Client{URL: fmt.Sprintf("%s/usage_events", url), httpc: c}
+type Client interface {
+	Send(ctx context.Context, payload []byte, traceID TraceID) error
```

Comment:
IIUC: Generally the `Send` methods are not intended to be used externally, and exist only so that `billing.Send` can be generic.

WDYT about making it private? (and adding a comment noting that it's just the implementation-specific part of `Send[E]()`)